rotkehlchen/chain/manager.py

```diff
@@ -771,23 +793,33 @@ def query_polkadot_balances(
             total_balance += balance
         self.totals.assets[A_DOT] = total_balance
 
-    def sync_btc_accounts_with_db(self) -> None:
-        """Call this function after having deleted BTC accounts from the DB to
+    def sync_btc_accounts_with_db(
```

Comment:
rename to make it obvious it's for all bitcoin chains.

```suggestion
    def sync_bitcoin_accounts_with_db(
```